doc/getting-started.md

```diff
@@ -312,12 +328,15 @@ req id=0:2902 src=10.1.8.151:51978 dst=10.1.8.150:80 :method=GET :authority=web-
 ...
 ```
 
-___
+____
 
 ## That’s it! 👏
-For more information about Conduit, check out the
-[overview doc](/docs) and the [roadmap doc](/roadmap), or hop into the #conduit channel on [the
-Linkerd Slack](https://slack.linkerd.io) or browse through the
+
+For more information about Linkerd, check out the
+[overview doc](/docs) and the [roadmap doc](/roadmap), or hop into the #linkerd2
```

Comment:
Should we go ahead and rename the #conduit channel in the linkerd slack to match? If not (see other comment), let's change this back to #conduit.